main_app/templates/home.html

```diff
@@ -3,36 +3,70 @@
 {% block content %}
 
 {% if user.is_authenticated %}
-<div class="col-md-6">
-  <h1>Welcome to IVM</h1>
-</div>
+<h3 class="m-4">All Items in your inventory</h3>
+<div class="row">
+  <div class="col-12">
+    {% if items %}
+    <div class="card bg-light ms-4 me-4 mb-4">
+      <div class="card-header">
+        <i class="fa-solid fa-list fa-lg"></i> Item Records
+      </div>
+      <div class="card-body">
+        <p class="card-text">
+          <div class="table-responsive ">
+            <table class="table table-hover">
+              <thead>
+                <tr>
+                    <th scope="col">ID</th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Type</th>
+                    <th scope="col">Description</th>
```

Comment:
Was thinking of having the order of these columns be a little different.
ID | Name | Type | Count | Stock Status | Edit | Delete

Stock status is a conditional to show the level of the quantity_current compared to the quantity_min to make sure that the users can quickly see what needs to be restocked.